### ISSUE TITLE
CCI creates scratch org with SFDX alias in quotes

### ISSUE THREAD

**Issue by dospromptman** [SalesforceFoundation/CumulusCI#664]
It used to be that the scratch org name given to CCI would get passed along to sfdx to use as the alias when creating the scratch org.  It was possible to use the alias with the project name prefixed to conveniently invoke sfdx against a scratch org.

However, the aliases are now created with quotes, such that the alias name is surrounded in literal quote characters.  This requires escaping the quotes when trying to use sfdx with that alias.

This seems like a bug to me.  Why is it wrapping it in quotes now?  Why aren't the quotes getting removed from the argument when the sfdx command is invoked from cci?

```
cci org scratch dev itchy
cci org info itchy
```

This should work, but it does not:

```
sfdx force:org:display -u SAL__itchy
```

Instead, only this works:

```
sfdx force:org:display -u \"SAL__itchy\"
```

See attached screenshot.

<img width="974" alt="cci_alias_quotes_bug" src="https://user-images.githubusercontent.com/1058323/41166864-ee114744-6b06-11e8-88ae-2ed976f2b1c0.png">

**Comment by force2b** [comment]
I noticed this yesterday, but wasn't sure if it was me and haven't had a chance to get back to it. Thanks for documenting and posting!

**Comment by cdcarter** [comment]
Oh well thats annoying!

**Comment by jlantz** [comment]
Fixed in 2.0.4
